services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/organisaatio/Persistence.kt

```diff
@@ -0,0 +1,19 @@
+package fi.hel.haitaton.hanke.organisaatio
+
+import org.springframework.data.jpa.repository.JpaRepository
+import java.time.LocalDateTime
+import javax.persistence.*
+
+@Entity @Table(name = "organisaatio")
+class OrganisaatioEntity(
+    var organisaatioTunnus: String? = null,
+    var nimi: String? = null,
+    var createdAt: LocalDateTime? = null,
+    var modifiedAt: LocalDateTime? = null,
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Int? = null
+)
+
+interface OrganisaatioRepository : JpaRepository<OrganisaatioEntity, Int> {
+    fun findAllByOrderByIdAsc(): Iterable<OrganisaatioEntity>
```

Comment:
Out of curiosity: why we order by id? I would maybe have chosen ordering by nimi or organisaatioTunnus if ordering needs to be done in this level.

Reply:
Yep, changed it to order by nimi.